pkg/services/mysqldb/bind.go

```diff
@@ -7,29 +7,113 @@ import (
 	"github.com/Azure/open-service-broker-azure/pkg/service"
 )
 
-func (s *serviceManager) ValidateBindingParameters(
+func (a *allInOneManager) ValidateBindingParameters(
 	bindingParameters service.BindingParameters,
 ) error {
 	// There are no parameters for binding to MySQL, so there is nothing
 	// to validate
 	return nil
 }
 
-func (s *serviceManager) Bind(
+func (v *vmOnlyManager) ValidateBindingParameters(
```

Comment:
This matches what we did in sqldb, but I think we should change the name of this manager (and the service it manages) here (and, separately, in the sqldb module as well). "VM" implies some things that might not actually be true for certain plans.

Ok... we can take an educated guess that there are VMs involved (currently), but in the future, these might be containerized and the end-user would be none the wiser. Also, it's possible (assuming support for geo-redundancy, master/slave, etc.) that there are _multiple_ VMs/containers involved.

Let me know if you hate this name, but I think the right nomenclature to opt for here might be `dbmsOnlyManager`. I feel as if that will accurately convey what it is we're managing without implying anything about how that's realized.

Reply:
That's good with me